src/Collapse/CollapseGroup/CollapseGroup.component.tsx

```diff
@@ -0,0 +1,83 @@
+// REACT
+import * as React from 'react';
+// VENDOR
+import classNames from 'classnames';
+import styled from 'styled-components';
+// COMPONENTS
+import { Themes } from '../Collapse.component';
+
+// The mundane function of CollapseGroup is to remove the bottom border of any Collapse children,
```

Comment:
Would this be worth a unit test?

Reply:
It took me a bit to figure out how to do this. Here's an example from `Badge` that may help:
https://github.com/RetailMeNot/anchor/blob/master/src/Badge/Badge.component.spec.tsx#L57

and you need `jest-styled-components` (which will also change your snapshots)

Reply:
I think that `jest-styled-components` has a hard time interpreting pseudo selectors. I used a fun one, `.anchor-collapse:not(:last-child)` and it can't seem to recognize the style even though in the browser I can see that it's working. This is something I would like to test for, but I'm wondering if it's possible.

Reply:
Ah yes, Enzyme might not support pseudo selectors yet. 

Would these work for finding the first and last ones?
```
find('.anchor-collapse').first()
find('.anchor-collapse').last()
```

Reply:
If that doesn't work, here is another possibility I found (less elegant but might still be useful):

```
find({ key: 'collapse-child-0' })
find({ key: 'collapse-child-2' }); 
```
from here: https://airbnb.io/enzyme/docs/api/selector.html#4-object-property-selector

Reply:
No matter what I did I couldn't get enzyme to play ball with this. I refactored the code to give `Collapse` a `hasBottomBorder` prop which adds a class to hide the border. Seeing as how that's the primary reason to have `CollapseGroup` exist it seemed prudent to have a test that actually checked it.